workflows/offline_ml_diags/offline_ml_diags/_plot_input_sensitivity.py

```diff
@@ -78,3 +81,88 @@ def plot_jacobian(model: fv3fit.keras._models.DenseModel, output_dir: str):
         plt.tight_layout()
         with fsspec.open(os.path.join(output_dir, LINE_NAME), "wb") as f:
             fig.savefig(f)
+
+
+def plot_rf_feature_importance(
+    input_feature_indices: Mapping[str, Tuple[int, int]],
+    wrapped_model: SklearnWrapper,
+    output_dir: str,
+) -> None:
+
+    importances = []
+    for member in wrapped_model.model.regressors:
+        importances.append(member.feature_importances_)
+    mean_importances = np.array(importances).mean(axis=0)
+    std_importances = np.array(importances).std(axis=0)
+
+    n_vector_features = len(
+        [
+            var
+            for var in input_feature_indices
+            if ((input_feature_indices[var][1] - input_feature_indices[var][0]) > 1)
+        ]
+    )
```

Comment:
Might be simpler to grok with a simpler loop
```
n_vector_features = 0
for var in ...:
    feature_len = 
    if feature_len > 1:
        n_vecture_features += 1
```

Reply:
Since we're already grabbing the lengths and making a determination of scalar or vector, we could separate things out up here and pass them into their respective functions instead of redoing the separation in each individual helper.